src/rfbzero/degradation.py

```diff
@@ -273,5 +273,62 @@ def degrade(self, c_ox: float, c_red: float, timestep: float) -> tuple[float, fl
             c_ox, c_red = mechanism.degrade(c_ox, c_red, timestep)
         return c_ox, c_red
 
-# TODO
-# def potential_dependent(potential, c_ox, c_red, timestep, rate=0):
+
+class Dimerization(DegradationMechanism):
+    """
+    Provides a reversible dimerization mechanism:
+        ox + red <--> dimer
+
+    Parameters
+    ----------
+    forward_rate_constant : float
+        Second order rate constant for forward reaction (1/M/s).
+    backward_rate_constant : float
+        First order rate constant for backward reaction (1/s).
+    c_dimer : float
+        Initial concentration of dimer, defaults to 0 (M).
+        Continuously updated.
```

Comment:
Probably don't need the `Continuosly update` line, that's not the concern of the parameter

Reply:
done